README.md

```diff
@@ -1,7 +1,77 @@
+
+
 # osDFS-Chatbot
-A community-driven bot that aims to be as simple as possible to serve humans queries about osDFS.
+A community-driven bot that aims to be as simple as possible to serve humans queries about osDFS.It is a machine-learning based conversational dialog engine build in Python which makes it possible to generate responses based on collections of known conversations.
+
+
+# 🚩 Vision
+- To build a conversational AI chatboT.
+- Learn the basics of chatbot development.
+
+
+## Getting Started
+- **Fork** this repository (Click the 'Fork' button, top right of this page)
+- **Clone** your forked copy of this repo down to your local machine:
+```
+git clone https://github.com/<your-github-username>/osDFS-Chatbot.git
+```
+- **Change directory** to get inside repository:
+```
+cd osDFS-Chatbot
+```
+- **Create a branch** for the files:
+```
+git checkout -b feature/<name-of-algorithm>
+```
+- *Or* if its a bugfix to a file:
+```
+git checkout -b bugfix/<title-of-issue>
+```
+- Make your changes locally
+- **Stage** and **commit** changes:
+```
+git add .
+git commit -m 'commit message'
+```
+- **Push** changes to GitHub:
+```
+git push origin <branch-name>
+```
+- Create a **New Pull Request** (PR) from your forked repository on GitHub (Click the 'New Pull Request' Button located at the top of your repo)
+
+## Useful Resources
+
+- [Tensorflow Docs](https://www.tensorflow.org/api_docs)
+- [Chatbot Basics](https://blog.tensorflow.org/2019/05/transformer-chatbot-tutorial-with-tensorflow-2.html)
+- [Git and GitHub](https://www.digitalocean.com/community/tutorials/how-to-use-git-a-reference-guide)
+### :warning: Issues
+
+Always feel free to **file a new issue** with a respective title and description on the [osDFS Chatbot](https://github.com/osDFS/osDFS-Chatbot/issues) repository.
+
+### :handshake: Contribution
+
+If you have any great ideas which can make this project more better, you can make changes and send us a Pull Request(PR) with a respective title and description on the [osDFS-Chatbot](https://github.com/osDFS/osDFS-Chatbot) repository, We will definitely review your pull request.
+
+### Project Admin
+
+| ![](https://avatars3.githubusercontent.com/u/61171342?s=200&v=4) |
+| :------------------------: |
+| **OPEN SOURCE DIGITAL FREEDOM SOCIETY**  |
+
+[![GitHub followers](https://github.com/osDFS) [![Twitter Follow](https://twitter.com/dfs_os)
+
+[![ForTheBadge built-with-love](http://ForTheBadge.com/images/badges/built-with-love.svg)](https://GitHub.com/akhilaku/)
+
+
+
+
 
-- Join our Slack channel and then search for `#osdfs-chatbot`
```

Comment:
I think it should be in the starting. People can find it easily.

Reply:
sure thing